ultramodern/include/ultramodern/renderer_context.hpp

```diff
@@ -33,14 +35,8 @@ namespace ultramodern {
             DWORD thread_id = (DWORD)-1;
             auto operator<=>(const WindowHandle&) const = default;
         };
-#elif defined(__ANDROID__)
-        using WindowHandle = ANativeWindow*;
-#elif defined(__linux__)
-        struct WindowHandle {
-            Display* display;
-            Window window;
-            auto operator<=>(const WindowHandle&) const = default;
-        };
+#elif defined(__linux__) || defined(__ANDROID__)
```

Comment:
Let's have a TODO here to restore a native window handle path, we're just skipping it for now to avoid a complicated config hierarchy.